Mathlib/Algebra/Homology/ShortComplex/Basic.lean

```diff
@@ -21,8 +20,10 @@ namespace CategoryTheory
 
 open Category Limits
 
-variable (C D : Type*) [Category C] [Category D]
+variable {C D E : Type*} [Category C] [Category D] [Category E]
+  [HasZeroMorphisms C] [HasZeroMorphisms D] [HasZeroMorphisms E]
 
+variable (C) in
 /-- A short complex in a category `C` with zero morphisms is the datum
 of two composable morphisms `f : X₁ ⟶ X₂` and `g : X₂ ⟶ X₃` such that
 `f ≫ g = 0`. -/
```

Comment:
I can't comment on the next line but can you remove `[HasZeroMorphisms C]`? Thanks.